rascal-vscode-extension/src/RascalExtension.ts

```diff
@@ -52,9 +53,11 @@ export class RascalExtension implements vscode.Disposable {
         this.registerTerminalCommand();
         this.registerMainRun();
         this.registerImportModule();
+        this.registerDebuggerCommand();
```

Comment:
I think the `RascalDebugViewProvider` should be responsible for registring this command, as without it, it doesn't function.

Reply:
Fair point. Updated.